.credo.exs

```diff
@@ -162,7 +162,7 @@
              ~w/shortdoc moduledoc behaviour use import alias require module_attribute defstruct callback public_fun private_fun/a,
            ignore: ~w/callback_impl/a
          ]},
-        {Credo.Check.Consistency.MultiAliasImportRequireUse, []},
+        {Credo.Check.Consistency.MultiAliasImportRequireUse, false},
```

Comment:
Why we need to change credo rule?

Reply:
@Nihisil
I think the `MultiAliasImportRequireUse` is not matched our [convention](https://nimblehq.co/compass/development/code-conventions/elixir/#modules), so I decide to disable it. Correct me if I'm wrong.

![Screenshot 2022-12-05 at 16 56 58](https://user-images.githubusercontent.com/17830319/205608186-1d710fc3-2708-4615-866c-ba9de7a5d46a.png)

TBH, I checked other ICs and saw we usually exclude our new files (e.g Accounts, etc.), or we disable this rule like I'm doing.

![Screenshot 2022-12-05 at 17 04 02](https://user-images.githubusercontent.com/17830319/205609683-8c7cd520-d751-4f93-a85b-cab979e753ce.png)

Reply:
In our client project, we have this

![image](https://user-images.githubusercontent.com/11751745/205815363-5d9506bc-1183-4275-821b-0c30025c013b.png)

Reply:
@andyduong1920 As a suggestion, can we ignore these warnings in the `Consistency` section?
Please correct me if I'm wrong in using `MultiAlias`

![Screenshot 2022-12-06 at 12 11 08](https://user-images.githubusercontent.com/17830319/205821876-5f612911-88e5-4b55-adc7-e20225ce530a.png)

Reply:
Yeah, this rule a bit tricky and it doesn't work good with generated code, but once you have your code it will be helpful to have this rule enabled.

You can check how Top handled these warnings: https://github.com/topnimble/elixir-internal-certification/blob/4f3c31216fbfad0bef1fc63a365d90c63f52d43c/.credo.exs#L165

Reply:
@Nihisil Thanks, I follow Top and update this rule in [fd075d7](https://github.com/vnntsu/elixir-ic/pull/31/commits/fd075d70d5c089e2aea1fca765c0995d2e6ef1b1)

Reply:
Should we add that config into the Template @Nihisil @vnntsu 😄

Reply:
Opened a PR on https://github.com/nimblehq/elixir-templates/pull/301